guides/common/modules/proc_synchronizing-content-over-a-network.adoc

```diff
@@ -0,0 +1,85 @@
+[id="synchronizing-content-over-a-network_{context}"]
+= Synchronizing Content Over a Network
+
+Configure a downstream {ProjectServer} to synchronize repositories from a connected upstream {ProjectServer} over HTTPS.
+
+.Prerequisites
+* A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.
+* You have imported the subscriptions manifest on both the upstream and downstream {ProjectServer}.
+* The required repositories are enabled for the organization on the upstream {ProjectServer}.
+* You have user permissions to create product repositories and organizations on the downstream {ProjectServer}.
+
+.Procedure
+. Navigate to *Content* > *Subscriptions*.
+. Click the *Manage Manifest* button.
+. Navigate to the *CDN Configuration* tab.
+. Select the *Upstream {ProjectServer}* tab.
+. In the *URL* field, enter the address of the upstream {ProjectServer}.
+. In the *Username*, enter your username for upstream login.
+. In the *Password*, enter your password or personal access token for upstream login.
+. In the *Organization label* field, enter the label of the upstream organization.
+. Optional: In the *Lifecycle Environment Label* field, enter the label of the upstream lifecycle environment.
+Default is `Library`.
+. Optional: In the *Content view label* field, enter the label of the upstream Content View.
+Default is `Default_Organization_View`.
+. From the *SSL CA Content Credential* menu, select a CA certificate used by the upstream {ProjectServer}.
```

Comment:
I usually get to this point and then lose all my work because I have to go create a content credential. I think it would be helpful to remind users before starting these steps to

1. Grab the certificate from the upstream Satellite.  This is available from `https://<upstream-satellite>/pub/katello-server-ca.crt`, or just grab the file `/etc/pki/katello/certs/katello-server-ca.crt`.
2. On the downstream Satellite, use this file to create a Content Credential. (Content > Content Credentials > Create Content Credential)

Now, it will be available in the list here.